config/jobs/kubernetes-sigs/cluster-api-provider-digitalocean/cluster-api-provider-digitalocean-periodics.yaml

```diff
@@ -1,4 +1,34 @@
 periodics:
+- name: periodic-cluster-api-provider-digitalocean-janitor
+  decorate: true
+  decoration_config:
+    timeout: 1h
+  interval: 12h
+  labels:
+    preset-dind-enabled: "true"
+    preset-kind-volume-mounts: "true"
```

Comment:
Tiny nit: as far as I understand, we run janitor as a (Go) binary. Do we need those two presets in that case?

Reply:
that is true, thanks for the catch!
@xmudrii PTAL